Core.Telemetry/OpenTelemetryMetric.cs

```diff
@@ -41,9 +41,14 @@ private static MeterProvider BuildMeterProvider()
                     }).Build();
 
                     break;
-                default:
+
+                case OpenTelemetryHelpers.ConsoleExporterName:
                     meterProvider = Sdk.CreateMeterProviderBuilder().AddMeter(SUBSCRIPTION_KEY).AddConsoleExporter().Build();
                     break;
+
+                default:
+                    meterProvider = Sdk.CreateMeterProviderBuilder().AddMeter(SUBSCRIPTION_KEY).Build();
```

Comment:
You could make this code nicer by moving common code of all switch branches above or below, respectively.
`Sdk.CreateMeterProviderBuilder().AddMeter(SUBSCRIPTION_KEY)` above, `return builder.Build();` below, or some such. An if-elseif-else might be simpler than the switch?

Reply:
I have some future plans to add additional exporters here so which I think will better lend itself to the switch pattern